backend/sim/wallet/wallet_internal_test.go

```diff
@@ -24,36 +24,19 @@ import (
 
 	"perun.network/go-perun/wallet/test"
 	"perun.network/go-perun/wire/perunio"
+	wiretest "perun.network/go-perun/wire/test"
+
 	pkgtest "polycry.pt/poly-go/test"
 )
 
-// TestSignatureSerialize tests serializeSignature and deserializeSignature since
-// a signature is only a []byte, we cant use io.serializer here.
-func TestSignatureSerialize(t *testing.T) {
-	a := assert.New(t)
-	// Constant seed for determinism
+func Test_Sig_GenericMarshaler(t *testing.T) {
```

Comment:
- Please try to adhere to function naming conventions. Underscore should be avoided.
https://ieftimov.com/post/testing-in-go-naming-conventions/#function-naming-conventions

- Please add function documentation.
```suggestion
// TestSignatureMarshalling tests marshalling and unmarshalling for type `Sig`.
func TestSignatureMarshalling(t *testing.T) {
```

Reply:
You could make this a black box test if you create a constructor for `Sig`.